### ISSUE TITLE
pontocerebellar hypoplasia

### ISSUE THREAD

**Issue by RiyaUthup** [obophenotype/human-phenotype-ontology#10838]
**Preferred term label:**pontocerebellar hypoplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**39082157

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@RiyaUthup 
We are trying to move away from "bundled" terms that describe more than one thing at a time. Can you use the combination of Pontine hypoplasia and Cerebellar hypoplasia?

**Comment by RiyaUthup** [comment]
Sure, I will use the combination of Pontine hypoplasia and Cerebellar hypoplasia for my annotation. Thank you!

**Comment by pnrobinson** [comment]
thanks!
